docs/content/installation/install_compatibility_requirements.md

```diff
@@ -17,11 +17,11 @@ For IBM Cloud® Satellite users, see [cloud.ibm.com/docs/satellite](https://clou
 
         Port 7778
 
-    -   **IBM Spectrum® Virtualize Family includes IBM® SAN Volume Controller and IBM FlashSystem® family members that are built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)**
+    -   **IBM Spectrum® Virtualize family includes IBM® SAN Volume Controller and IBM FlashSystem® family members that are built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)**
 
         Port 22
 
-    -   **DS8000® Family systems**
+    -   **DS8000® family systems**
 
         Port 8452
 
```

Comment:
I can't comment on the line below (`Be sure that multipathing is installed and running.`), but isn't it included in `1.`?

Reply:
multipathing is for the storage system in general. it certainly doesn't hurt to keep it in.
All of the other information is specifically per worker node.

Reply:
but "installed and running" is specifically per worker node, it's not about the storage system, so it sounds exactly the same to me

Reply:
in any case, it's something important that they do before connecting to csi so it's good that we keep it there, IMO

Reply:
but we already have this, in the same page

Reply:
after looking again. you are correct.
done

Reply:
now that the second bullet is gone, do we still need the first black dot?
![image](https://user-images.githubusercontent.com/57443811/146158570-14efc72a-faea-44b2-9636-1456c1c7b3a6.png)

Reply:
done